<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefNetwork</class>
 <widget class="QWidget" name="PrefNetwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>294</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="youtube_page">
      <attribute name="title">
       <string>&amp;Youtube (and other sites)</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="yt_support_check">
         <property name="text">
          <string>&amp;Enable Youtube internal support</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="youtube_widget" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="_3">
            <item>
             <widget class="QLabel" name="yt_label">
              <property name="text">
               <string>Playback &amp;quality</string>
              </property>
              <property name="buddy">
               <cstring>yt_quality_combo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="yt_quality_combo"/>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="yt_user_agent_label">
              <property name="text">
               <string>&amp;User agent</string>
              </property>
              <property name="buddy">
               <cstring>yt_user_agent_edit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yt_user_agent_edit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="streaming_check">
         <property name="text">
          <string>Enable &amp;MPV's support for streaming sites</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="proxy_page">
      <attribute name="title">
       <string>&amp;Proxy</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="use_proxy_check">
         <property name="text">
          <string>&amp;Enable proxy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="proxy_container" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Host:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_hostname_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="proxy_hostname_edit"/>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&amp;Port:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>proxy_port_spin</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QSpinBox" name="proxy_port_spin">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&amp;Username:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_username_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="4">
           <widget class="QLineEdit" name="proxy_username_edit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Pa&amp;ssword:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_password_edit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="4">
           <widget class="QLineEdit" name="proxy_password_edit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&amp;Type:</string>
            </property>
            <property name="buddy">
             <cstring>proxy_type_combo</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="proxy_type_combo"/>
          </item>
          <item row="3" column="2" colspan="3">
           <spacer>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>261</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tab_widget</tabstop>
  <tabstop>yt_support_check</tabstop>
  <tabstop>yt_quality_combo</tabstop>
  <tabstop>yt_user_agent_edit</tabstop>
  <tabstop>streaming_check</tabstop>
  <tabstop>use_proxy_check</tabstop>
  <tabstop>proxy_hostname_edit</tabstop>
  <tabstop>proxy_port_spin</tabstop>
  <tabstop>proxy_username_edit</tabstop>
  <tabstop>proxy_password_edit</tabstop>
  <tabstop>proxy_type_combo</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>use_proxy_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxy_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yt_support_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>youtube_widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
